app/(api)/_datalib/media/create.ts

```diff
@@ -0,0 +1,19 @@
+import { getClient } from '@utils/mongodb/mongoClient.mjs';
+
+export async function createMediaItem(inputData: object) {
+  //TODO: QUESTION is type of inputData now okay, if not could you please specify its exact type
+  try {
+    const client = await getClient();
+    const db = client.db;
+    const reqCollection = await db.getCollection('Media');
+    const reqDocument = await db.collection(reqCollection).insertOne(inputData);
+    return { ok: true, body: reqDocument, error: null };
+  } catch (error) {
+    console.error('error creating media item', error);
+    return {
```

Comment:
you can just have it print out the error msg rather than printing basically an empty error statement. I believe default errors you can just print error itself as the message component.